WORKSPACE

```diff
@@ -18,7 +18,7 @@ tf_workspace()
 
 # Specify the minimum required bazel version.
 load("//tensorflow:tensorflow.bzl", "check_version")
-check_version("1.4")
+check_version("0.14")
```

Comment:
this is probably also wrong.  I didn't know anyone was actually on bazel master, let me revert my original change :)